XenAdmin/Wizards/ExportWizard/ExportSelectVMsPage.cs

```diff
@@ -120,7 +120,7 @@ protected override void PageLoadedCore(PageLoadedDirection direction)
         protected override void PageLeaveCore(PageLoadedDirection direction, ref bool cancel)
 		{
 			if (direction == PageLoadedDirection.Forward && IsDirty)
-					cancel = !PerformCheck(CheckSpaceRequirements);
+					cancel = !PerformCheck(CheckSpaceRequirements, CheckDiskSizeForTransfer);

```

Comment:
I think CheckDiskSizeForTransfer should be the first check to run: if CheckSpaceRequirements fails, there is no point in looking for more space only to find out afterwards that we are still be unable to export.